src/main/scala/uk/gov/ons/bi/dataload/reader/LinksParquetReader.scala

```diff
@@ -15,4 +15,7 @@ class LinksParquetReader (ctxMgr: ContextMgr)
     spark.read.parquet(srcFilePath)
```

Comment:
The rename of this class is a bad smell and tells me the code isnt flexible. You should be working with interfaces that can be swappable. For example, take a reader interface and then pass in a jsonReader or parquetReader.